rules/st2workroom_st2installer_st2enterprise_test_master_ubuntu14.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
similar rule needs to be auto-generated for versioned release pipeline. It would have to be added in the prepare_release workflow.

Can be a separate PR.